x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -45,10 +45,16 @@ export class HeadlessChromiumDriver {
   private readonly inspect: boolean;
   private readonly networkPolicy: NetworkPolicy;
 
```

Comment:
Small nit: this code block probably doesn't need two flags since they're both get attached in the same call. Perhaps `private listenersAttached: boolean` and then you can just check it once in the open URL call.

Reply:
Absolutely -- great idea